samples/excel/85-preview-apis/chart-data-source.yaml

```
@@ -0,0 +1,129 @@
+order: 6
+id: excel-chart-data-source
+name: Chart series data source
+description: This sample shows how to get information about the data source of a chart series.
+host: EXCEL
+api_set:
+    ExcelApi: BETA (PREVIEW ONLY)
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#log-chart-series-source").click(() => tryCatch(logChartSeriesSource));
+
+        async function logChartSeriesSource() {
+          await Excel.run(async (context) => {
+            // This method retrieves the data source string and type values from
+            // the chart on the Sample worksheet.
+            const sheet = context.workbook.worksheets.getItem("Sample");
+
+            // Get the series collection from the first chart on the worksheet, and
+            // then get the first series in that collection.
```

Consider reframing this to be about the goal, rather than the step-by-step.
```suggestion
            // Get the first chart series from the first chart on the worksheet.
```